src/dmd/cli.d

```diff
@@ -829,7 +827,6 @@ dmd -cov -unittest myprog.d
     static immutable reverts = [
         Feature("dip25", "useDIP25", "revert DIP25 changes https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md"),
         Feature("intpromote", "fix16997", "revert integral promotions for unary + - ~ operators"),
-        Feature("markdown", "markdown", "disable Markdown replacements in Ddoc"),
```

Comment:
Should reverts be deprecated first?

Reply:
Dunno. Part of this PR was to figure out what, if anything, breaks. Buildkite is green so probably not much. 
If this were a switch that affected codegen I'd be much more cautious, but this affects documentation only, and it's been enabled by default for several _years_ now.